### DISCUSSION TITLE
Live For Desktop

### DISCUSSION THREAD

**Discussion by xTye** [tidal-music/discussions#34] [Ideas & Feature requests]
One big feature that would be nice to have is Tidal Live for desktop users. My friends and I play music together when we game and we often have to use Watch2Gether; unfortunately it doesn't have great things like Tidal's recommendation algorithms. Instead, we have to queue every song individually which can get quite annoying.

Additionally, it would be nice to have a feature that allows other people in the live session to queue songs.

Perhaps at a more granular level of understanding it would be nice to have:

- Live sessions on desktop
- When items aren't queue'd recommendation songs play (like when a playlist / album ends)
- Have other members queue / play songs (recommendation songs are then updated)

And some stretch goals:

- Permissions for members
- Discord integrations

EDIT: separate stretch goals

**Comment by stefanmoro** [comment]
Hey @xTye 

This GitHub discussions forum aims at handling technical discussions around the [TIDAL Developer Platform](https://developer.tidal.com) (TIDAL API, open source projects etc).

As this question relates to an existing product feature, I will kindly ask you to reach out to our main [TIDAL support](https://support.tidal.com/).

Thanks
